protocols/direct-fund/direct-fund.go

```diff
@@ -254,6 +270,37 @@ func (s DirectFundObjective) clone() DirectFundObjective {
 
 	return clone
 }
+func (s DirectFundObjective) createSignedStateMessages(ss state.SignedState) []protocols.Message {
+
+	messages := make([]protocols.Message, 0)
+	for i, participant := range ss.State().Participants {
+
+		// Do not generate a message for ourselves
+		if uint(i) == s.C.MyIndex {
+			continue
+		}
+		message := protocols.Message{To: participant, ObjectiveId: s.Id(), SignedStates: []state.SignedState{ss}, Proposal: nil}
+		messages = append(messages, message)
+	}
+	return messages
+}
+
+func (s DirectFundObjective) signAndStore(toSign state.State, secretKey *[]byte) (state.SignedState, error) {
```

Comment:
Not affected by this PR, but I think we should rename the receiver variable to `o` for objective? `s` makes me think `state`...

Reply:
#209